### DISCUSSION TITLE
training about PowerPaint

### DISCUSSION THREAD

**Discussion by D222097** [open-mmlab/mmediting#2111] [Models]
Nice work!
I'd like to know more about the training.Will the training code be released later?
Besides, Part **3.3. Implementation Details** of the paper says:
> The main task and the text-to-image generation task have probabilities of 80% and 20%, respectively, in the training phase.

I tried to train a inpainting model in this way, but found the loss don't converge when text-to-image generation task is added. Don't know what I'm missing and be grateful if anyone could give me advice.

**Comment by Mengmengbai** [comment]
> Nice work! I'd like to know more about the training.Will the training code be released later? Besides, Part **3.3. Implementation Details** of the paper says:
> 
> > The main task and the text-to-image generation task have probabilities of 80% and 20%, respectively, in the training phase.
> 
> I tried to train a inpainting model in this way, but found the loss don't converge when text-to-image generation task is added. Don't know what I'm missing and be grateful if anyone could give me advice.

Could you share you training  PowerPaint code? I don't know how to train the follows:
Pobj for text-guided object inpainting
Pctxt for context-aware image inpainting
